pkg/controller/containerjfr/containerjfr_controller.go

```diff
@@ -232,14 +241,14 @@ func (r *ReconcileContainerJFR) Reconcile(request reconcile.Request) (reconcile.
 	}
 
 	exporterSvc := resources.NewExporterService(instance)
-	url, err = r.createService(context.Background(), instance, exporterSvc, &exporterSvc.Spec.Ports[0])
+	url, err = r.createService(context.Background(), instance, exporterSvc, &exporterSvc.Spec.Ports[0], openshiftv1.TLSTerminationReencrypt)
 	if err != nil {
 		return reconcile.Result{}, err
 	}
 	serviceSpecs.CoreAddress = url
 
-	cmdChanSvc := resources.NewCommandChannelService(instance)
-	url, err = r.createService(context.Background(), instance, cmdChanSvc, &cmdChanSvc.Spec.Ports[0])
+	cmdChanSvc := resources.NewCommandChannelService(instance) // TODO Remove?
+	url, err = r.createService(context.Background(), instance, cmdChanSvc, &cmdChanSvc.Spec.Ports[0], openshiftv1.TLSTerminationEdge)
```

Comment:
Should this have the same termination as the "exporter" service?

Reply:
It likely should be. I wasn't sure whether to remove the command channel route/service altogether at this point. What do you think?

Reply:
Yea, I'll think about if there's any reason to keep it around - but the command channel is now generally deployed as a WebSocket over the same port as the HTTP "exporter" service anyway, and I don't foresee us changing that, so a single service/route can probably cover both just fine.

Reply:
Sounds good. It could slightly simplify our deployment for now.